recipe/0001-remove-certifi-dependency.patch

```diff
@@ -0,0 +1,11 @@
+diff -ruN fiona-1.9.5/pyproject.toml fiona-1.9.5-patch/pyproject.toml
+--- fiona-1.9.5/pyproject.toml	2023-10-12 13:14:23.000000000 -0600
++++ fiona-1.9.5-patch/pyproject.toml	2023-11-28 10:47:25.666126879 -0700
+@@ -37,7 +37,6 @@
+ requires-python = ">=3.7"
+ dependencies = [
+     "attrs>=19.2.0",
+-    "certifi",
```

Comment:
Fiona imports certifi, so this is needed.

Reply:
See the discussion starting at https://github.com/conda-forge/fiona-feedstock/pull/220#issuecomment-1830275315
and in
https://github.com/conda-forge/fiona-feedstock/pull/188
The issue is that conda-forge only supports `certifi` for python versions where it's actually needed.  But your inclusion upstream of `certifi` for all python causes pip check to fail.  So I have to remove it manually from the `pyproject.toml` to allow `pip check` to pass on downstream packages.

This is actually the whole reason I got involved with maintaining Fiona in the first place.  I was trying to fix downstream `pip check`.

Reply:
I'll just try adding it back.  It seems like it *should* work for all python so I'm not sure what the previous issue was...